docs/INFRASTRUCTURE.md

```diff
@@ -0,0 +1,133 @@
+<!--
+Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+
+SPDX-License-Identifier: curl
+-->
+
+# Infrastructure in the curl project
+
+Overview of infrastructure we maintain, host and run in the project for the
+project.
+
+## git repository
+
+Since 2010, the main curl git repository has been hosted by GitHub, available
+at https://github.com/curl/curl.
+
+We also use the issue tracker, pull requests and discussions on GitHub.
+
+curl has an "enterprise account" on GitHub and is an "organization" on the
+site.
+
+We accept sponsorship via GitHub Sponsors.
+
+## CI services
+
+For every pull request and git push to the master repository, a number of
+build and testing jobs are run on a set of different CI services. The exact
+services vary over time. GitHub Actions and AppVeyor are the primary ones
+these days.
+
+## OSS-Fuzz
+
+Google runs the [OSS-Fuzz](https://google.github.io/oss-fuzz/) project which
+also runs fuzzing on curl code, non-stop, in their infrastructure and they
+send us emails in the rare instances they actually find something.
```

Comment:
can we clarify emails here (ex. to security@curl.se) ?

Reply:
OSS-Fuzz actually notifies those that are members in the "curl team" there. It requires a @gmail.com address and currently me, @icing and @cmeister2 are members. I'll add a mention about this.

If other maintainers want to get the emails (and access to the reports before they are made public after 90 days), just ping me.